roles/setup-services/tasks/configure-keycloak.yml

```diff
@@ -24,6 +24,38 @@
          keycloak_auth_response.status in [200, 401, 403]
   ignore_errors: yes
 
+- name: Fetch public client JSON from keycloak
+  uri:
+    url: "https://{{ keycloak_route_output.stdout }}/auth/admin/realms/{{namespace}}/clients/{{PUBLIC_CLIENT_ID}}"
+    method: GET
+    validate_certs: no
+    body_format: json
+    headers:
+      Authorization: "Bearer {{ keycloak_auth_response.json.access_token }}"
+    status_code: [200]
+  register: public_client_original
+  ignore_errors: yes
+
+- name: Write keycloak client JSON to file
+  copy: content="{{public_client_original.json}}" dest=/tmp/keycloak-client-original.json
+
+- name: Create new keycloak client JSON with updated webOrigins & redirectUris
+  command: >
+    jq '.webOrigins=["ionic://localhost","http://localhost","http://localhost:8100"] | .redirectUris=["http://localhost:*", "http://localhost"]' /tmp/keycloak-client-original.json
```

Comment:
From testing on iOS and Android these are the required settings. I considered adding them to `aerogearcatalog/keycloak-apb`, but seems it is better here since it is specific to ionic-showcase.

Reply:
So is `jq` installed by default on bastion servers? Should we mention in the readme that `jq` should be installed prior running this playbook?

Reply:
@psturc yep, just verified on a PDS instance. Do you think we should add a check for it in this setup?

Reply:
Yes, having this check as a first task in setup-services role would be nice 👍

Reply:
@psturc added the check for `jq` 👍